pkg/template/release.go

```diff
@@ -9,17 +9,17 @@ import (
 
 func BuildImages(registryDomain string, versions Versions) Images {
 	return Images{
-		CalicoCNI:                    buildImage("giantswarm/cni", versions.Calico),
-		CalicoKubeControllers:        buildImage("giantswarm/kube-controllers", versions.Calico),
-		CalicoNode:                   buildImage("giantswarm/node", versions.Calico),
-		Etcd:                         buildImage("giantswarm/etcd", versions.Etcd),
-		Hyperkube:                    buildImage("giantswarm/hyperkube", versions.Kubernetes),
-		KubeApiserver:                buildImage("giantswarm/kube-apiserver", versions.Kubernetes),
-		KubeControllerManager:        buildImage("giantswarm/kube-controller-manager", versions.Kubernetes),
-		KubeProxy:                    buildImage("giantswarm/kube-proxy", versions.Kubernetes),
-		KubeScheduler:                buildImage("giantswarm/kube-scheduler", versions.Kubernetes),
-		KubernetesAPIHealthz:         buildImage("giantswarm/k8s-api-healthz", versions.KubernetesAPIHealthz),
-		KubernetesNetworkSetupDocker: buildImage("giantswarm/k8s-setup-network-environment", versions.KubernetesNetworkSetupDocker),
+		CalicoCNI:                    buildImage("giantswarm/cni", versions.Calico, ""),
+		CalicoKubeControllers:        buildImage("giantswarm/kube-controllers", versions.Calico, ""),
+		CalicoNode:                   buildImage("giantswarm/node", versions.Calico, ""),
+		Etcd:                         buildImage("giantswarm/etcd", versions.Etcd, ""),
+		Hyperkube:                    buildImage("giantswarm/hyperkube", versions.Kubernetes, ""),
+		KubeApiserver:                buildImage("giantswarm/kube-apiserver", versions.Kubernetes, "-giantswarm"),
```

Comment:
I've looked into quay and didn't find this repo

Reply:
kube-apiserver? It's here https://quay.io/repository/giantswarm/kube-apiserver

Reply:
Ah, crap. `-giantswarm` is a tag prefix, not a repo prefix